src/content/reference/manifest.mdx

```diff
@@ -560,6 +560,47 @@ selector:
   app.kubernetes.io/name: vote
 ```
 
+### services ([object], optional)
```

Comment:
In the CLI 2.0, why do we need services, if you can do okteto up to multiple things at the same time? wouldn't it be easier for our customers to support `okteto up web worker`? (I'm mostly curious about the plan and how we differentiate one from the other for our customers)

Reply:
we don't support `okteto up web worker`. We did something similar with the `--detach` flag, but it's only available in the docker-desktop feature branch.
At some point we should have a better experience to activate development on several containers at the same time, but while we get there, `services` is only alternative to cover this scenario today